component/componenttest/shutdown_verifier.go

```diff
@@ -61,7 +62,7 @@ func verifyTracesProcessorDoesntProduceAfterShutdown(t *testing.T, factory compo
 }
 
 // VerifyProcessorShutdown verifies the processor doesn't produce telemetry data after shutdown.
-func VerifyProcessorShutdown(t *testing.T, factory component.ProcessorFactory, cfg component.Config) {
+func VerifyProcessorShutdown(t *testing.T, factory processor.Factory, cfg component.Config) {
```

Comment:
I think this needs to also be moved to `processortest`.

Reply:
Done